src/passes/Inlining.cpp

```diff
@@ -198,6 +224,14 @@ struct FunctionInfoScanner
     }
 
     info.size = Measurer::measure(curr->body);
+
+    if (auto* call = curr->body->dynCast<Call>()) {
+      if (info.size == call->operands.size() + 1) {
+        // This function body is a call with some trivial (size 1) operands like
+        // LocalGet or Const, so it is a trivial call.
```

Comment:
Counting by number of `Expression`s rather than by number of bytes, right?

Reply:
Yes, `Measurer` just counts Expressions. That happens to be useful here, but in general we might want to add something more accurate some day for bytes.